tests/cucumber/docker/run_docker.sh

```diff
@@ -7,7 +7,10 @@ rm -rf test-harness
 rm -rf tests/cucumber/features
 
 # clone test harness
-git clone --single-branch --branch master https://github.com/algorand/algorand-sdk-testing.git test-harness
+git clone --single-branch --branch devmodenet https://github.com/algorand/algorand-sdk-testing.git test-harness
+
+# Export env variable so network is set to dev mode
+export NETWORK_TEMPLATE="DevModeNetwork.json"
```

Comment:
@algochoi Minor:  Based on https://github.com/algorand/algorand-sdk-testing/pull/206/commits/16bf6edfe8a059b2ad745023ec145fb7f2b5dfa2, the override can be removed.

```suggestion
```